c7n/policy.py

```diff
@@ -638,6 +656,24 @@ def provider_name(self):
     def region(self):
         return self.data.get('region')
 
+    @property
+    def tz(self):
+        if 'tz' in self.data:
+            return tz.gettz(self.data.get('tz'))
+        return tz.gettz('UTC')
+
+    @property
+    def start(self):
+        if self.data.get('start'):
```

Comment:
what's the expected behavior if a user specifies start/stop with tz embedded, but not tz, per the current it would replace their explicit tz config with a default utc.

Reply:
At this point in time I'm leaning towards requiring non-offset dt's and using the tz attribute to explicitly set the tz each time-- otherwise default to UTC. Will update to reflect this

Reply:
simply ignoring tz doesn't feel great either, and leads back to the same issue, a user explicit's explicit config is ignored and replaced.